src/microsoft/office/license/user/microsoft-office-license-user.controller.js

```diff
@@ -35,25 +35,17 @@ angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicens
                     this.loaders.userPoll[item] = true;
 
                     this.license.pollUserDetails(this.$scope.currentLicense, item, this.$scope)
```

Comment:
I know it's not you, but ```this.$scope``` as a param... really?..

Reply:
services is asking for it. Soo that's it !